cmake/modules/openssl.cmake

```diff
@@ -22,7 +22,7 @@ else()
 			PREFIX "${PROJECT_BINARY_DIR}/openssl-prefix"
 			URL "https://github.com/openssl/openssl/archive/OpenSSL_1_0_2n.tar.gz"
 			URL_HASH "SHA256=4f4bc907caff1fee6ff8593729e5729891adcee412049153a3bb4db7625e8364"
-			CONFIGURE_COMMAND ./config shared --prefix=${OPENSSL_INSTALL_DIR}
+			CONFIGURE_COMMAND ./config no-shared --prefix=${OPENSSL_INSTALL_DIR}
```

Comment:
Why?

Reply:
It basically disables shared objects. I guess a static build is what we want when embedding openssl, isn't it?
I thought `shared` was used before to work around some compatibility issues we don't have anymore, so I switched that `no-shared`.